### PR TITLE
WAR-815 : runmode and retry support at suite level in project file

### PR DESCRIPTION
**Pull request by Annam16**
Added support for runmode and retry in test suites in project file

### CODE REVIEW

File: warrior/WarriorCore/project_driver.py [hunk 1]
```diff
@@ -217,21 +261,27 @@ def execute_project(project_filepath, auto_defects, jiraproj, res_startdir, logs
...
-                pj_junit_object.update_attr("impact", impact_dict.get(testsuite_impact.upper()),
+                pj_junit_object.update_attr("status", "SKIPPED", "ts",
+                                            tmp_timestamp)
+                pj_junit_object.update_attr("skipped", "1", "pj",
+                                            tmp_timestamp)
+                pj_junit_object.update_count("suites", "1", "pj",
+                                             tmp_timestamp)
+                data_repository['testsuite_{}_result'.format(suite_cntr)] = "SKIP"
+                # pj_junit_object.add_testcase_message(tmp_timestamp,
+                # "skipped")
```
Comment (kaheichan) [inline warrior/WarriorCore/project_driver.py]: Hi @Annam16 I believe this part is complained only by default pylintrc, please use our project pylintrc file to make format change

File: warrior/WarriorCore/project_driver.py [hunk 2]
```diff
@@ -241,42 +291,118 @@ def execute_project(project_filepath, auto_defects, jiraproj, res_startdir, logs
...
 
-        goto_testsuite_num = onerror_driver.main(testsuite, project_error_action, project_error_value)
-        if goto_testsuite_num == False:
+        goto_testsuite_num = onerror_driver.main(testsuite,
+                                                 project_error_action,
+                                                 project_error_value)
+        if goto_testsuite_num is False:
             onerror = "Next"
-        elif goto_testsuite_num == "ABORT":
+        elif goto_testsuite_num is "ABORT":
```
Comment (kaheichan) [inline warrior/WarriorCore/project_driver.py]: https://stackoverflow.com/questions/1504717/why-does-comparing-strings-in-python-using-either-or-is-sometimes-produce
Please refer to the above link, try avoid using is in this kind of string comparison

File: warrior/WarriorCore/project_driver.py (hunk 1 above)
Comment (Annam16) [inline warrior/WarriorCore/project_driver.py]: Sure @kaheichan

**Comment by bjrr** [comment]
@Annam16 Please correct the subject to follow the use convention. WAR-123: Subject text

File: warrior/WarriorCore/project_driver.py (hunk 2 above)
Comment (Annam16) [inline warrior/WarriorCore/project_driver.py]: Done

**Pull request merged**